zephyr/west.yml

```diff
@@ -0,0 +1,39 @@
+# The west manifest file for upstream Zephyr module ZXA.
+#
+# The per-installation west configuration file, .west/config, sets the
+# path to the project containing this file in the [manifest] section's
+# "path" variable.
+#
+# This imports the latest upstream Zephyr manifest file.
+#
+# You can pass your manifest repositories to west init when creating a
+# new Zephyr installation. See the west documentation for more
+# information.
+#
+# Beginning with west 0.8, the following command-line options
+# can be used to initialize with a repository lacking a west.yml
+# file at the top level.
+#
+#   west init
+#     -m <repository>
+#     --mr <branch_name>
+#     --mf <manifest_pathname>
+#     <workspace>
+
+manifest:
+  version: 0.7  # This file uses syntax from west 0.7
```

Comment:
But this file cannot be used as the manifest except with west 0.8, so it might be worth using that instead.

Reply:
I thought about that.  How is the `version` field value used by `west`?